backend/main/test/Group/Query/FindContainerQuery.test.ts

```diff
@@ -0,0 +1,123 @@
+import { afterEach, beforeEach, describe, expect, it, vi } from "vitest";
+import { FindContainerQuery } from "src/Group/Query/FindContainer/FindContanerQuery";
+import { UserId } from "src/User";
+import { UserIsNotAuthorized } from "src/Group/Query/FindContainer/IFindContainerUseCase";
+import { NishikiDynamoDBClient } from "src/Shared/Adapters/DB/NishikiTableClient";
+
+const USER_ID = UserId.generate();
+const GROUP_ID = "4898aa6e-ef03-4ae2-86c4-1db39b25a0bc";
+const GROUP_NAME = "group name";
+const CONTAINER_ID = "6112947e-e2bf-4a68-b3e5-9c39a937a297";
+const CONTAINER_NAME = "container name";
+
+describe("find container use case", () => {
+	let mockNishikiDynamoClient: NishikiDynamoDBClient;
+	let useCase: FindContainerQuery;
```

Comment:
Could you change this variable name to `query`? But it's not a big deal, so it's up to you.

Reply:
Thank you for pointing it out. I fixed it.